client/app/src/js/services/Screenshot.js

```diff
@@ -17,15 +17,13 @@ function Screenshot($http, User) {
 
   var addScreenshot = function (url) {
     var user = User.getUser();
-    console.log(user);
     return $http.post('/api/user/' + user.username + '/screenshot', {
       url: url
     });
   };
 
-  var getScreenshot = function(id){
-    var user = User.getUser();
-    return $http.get('/api/user/' + user.username + '/screenshot/' + id)
+  var getScreenshot = function(user, id){
+    return $http.get('/api/user/' + user + '/screenshot/' + id)
```

Comment:
Probably nice to record anticipated results. Namely, we're getting back a `Screenshot` model from DB. I saw this sometimes in Scott Moss's solution branch and found it really useful to follow the code more intuitively.